### PR TITLE
WRO-9768: Add scaled Item sample for Marquee qa-sampler

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Need sample to test Marquee for scaled item case.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add scaled Item sample for Marquee qa-sampler

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-9768

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: samples/sampler/stories/qa/Marquee.js [hunk 1]
```diff
@@ -386,3 +392,31 @@ WithTextCentered.parameters = {
 		hideNoControlsWarning: true
 	}
 };
+
+export const ScaledItem = (args) => {
+	return (
+		<MarqueeItem className={css.scaledItem} >
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Marquee.js:398]:
```suggestion
		<MarqueeItem className={css.scaledItem}>
```

File: samples/sampler/stories/qa/Marquee.js [hunk 2]
```diff
@@ -386,3 +392,31 @@ WithTextCentered.parameters = {
...
+
+select('alignment', ScaledItem, [null, 'left', 'right', 'center'], Marquee);
+select('forceDirection', ScaledItem, ['', 'ltr', 'rtl'], Marquee, '');
+number('marqueeDelay', ScaledItem, Marquee, 1000);
+boolean('marqueeDisabled', ScaledItem, Marquee, false);
+number('marqueeResetDelay', ScaledItem, Marquee, 1000);
+number('marqueeSpeed', ScaledItem, Marquee, 60);
+select('children', ScaledItem, mixedText, Marquee, mixedText[0]);
+
+ScaledItem.storyName = 'ScaledItem';
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Marquee.js:422]: How about `with Scaled Item`?

**Review by 0x64 — APPROVED** [review]

File: samples/sampler/stories/qa/Marquee.js (hunk 1 above)
Comment (jeonghee27) [reply samples/sampler/stories/qa/Marquee.js]: Thanks

File: samples/sampler/stories/qa/Marquee.js (hunk 2 above)
Comment (jeonghee27) [reply samples/sampler/stories/qa/Marquee.js]: I fixed

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**